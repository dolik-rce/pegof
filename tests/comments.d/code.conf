input comments.d/code.peg
