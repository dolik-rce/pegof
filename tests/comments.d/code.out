main <- "A"

# weirdly indented
# code comment
%%
int main() {
    printf("hello world!\n");
    return 0;
}
